install_scripts/templates/common/kubernetes-upgrade.sh

```diff
@@ -218,6 +218,17 @@ restartKubeproxy() {
     while kubectl -n kube-system get pods | grep kube-proxy | grep -v Running | grep -q kube-proxy; do
         sleep 1
     done
+    # wait for the iptables rule in the KUBE-SERVICES chain
+    while ! sudo iptables -L -t nat | grep -qE '\b10.96.0.1\b' ; do
```

Comment:
isnt this overridable?

https://github.com/replicatedhq/replicated-installer/blob/master/install_scripts/templates/kubernetes/init.sh#L39

Reply:
This is no longer needed since going back to IPVS